### DISCUSSION TITLE
Resize touchpad area in touchpad menu

### DISCUSSION THREAD

**Discussion by cassepipe** [linuxmint/discussions#27] [Ideas]
There is a command line tool for synaptics driven touchpad to resize its area like so : https://web.archive.org/web/20220927101543/http://tuxdiary.com/2014/12/16/reduce-effective-area-touchpad-linux/

But it'd be imho a great feature to be able to set the surface area with a bounding rectangle whose sides are pointer draggable, 
a bit like when you're trying to crop  a image. It would be non-dev friendly and would imho align with Cinnamon goals to be approachable yet configurable.

I know there is a "disable while typing" option but in effects it has not worked at all on my machines and I regularly get a jumping cursor.

What do you think ?

**Comment by badh0r53** [comment]
I'm not sure we are after the same thing, but I have added a very specific feature request that _may_ be related to the "cursor jumping" portion of your problem.

https://github.com/orgs/linuxmint/discussions/890
